layouts/partials/article-meta.html

```diff
@@ -1,5 +1,7 @@
 			<p>
 				{{ if not (eq (.Date.Format "2006-01-02") "0001-01-01") }}<time>{{ .Date.Format "2006-01-02" }}</time>{{ end }}
+{{ if default true (index .Site.Params "show-wordcount") }}
```

Comment:
I think we need to be stringent with naming of params. It should probably be `ShowWordCount`.